lib/stormpath-sdk/resource/base.rb

```diff
@@ -57,6 +57,10 @@ def non_printable_properties
       @non_printable_properties ||= []
       Array.new @non_printable_properties
     end
+
+    def has_status(name: :status, status_list: :default)
```

Comment:
It actually isn't a check - it's a ActiveRecord Rails like macro for defining the status getter and setter with default values. 

I will let the server respond with an error if you've set it to something invalid. Moving this.